content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -273,7 +273,7 @@ Use your preferred method to start Kubernetes cluster on Linux. Please note that
 ## Support for kubeadm join
 
 If your cluster has been created by [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/),
-and your networking is setup correctly using one of the methods listed above (networking is setup outside of kubeadm), you can use kubeadm to add a Windows node to your cluster. At a high level, you first have to initialize the master with kubeadm (Linux), then set up the CNI based networking (outside of kubeadm), and finally start joining Windows or Linux worker nodes to the cluster. For additional documentation and reference material, visit the kubeadm link above.
+and your networking is setup correctly using one of the methods listed above (networking is setup outside of kubeadm), you can use kubeadm to add a Windows node to your cluster. At a high level, you first have to initialize the master with kubeadm (Linux), then set up the CNI based networking (outside of kubeadm), and finally start joining Windows or Linux nodes to the cluster. For additional documentation and reference material, visit the kubeadm link above.
```

Comment:
`worker` is also a word that is not received very well, but i think we should still use it in places where the context demands it. worker nodes **do** have workloads.

having said that the above change is fine.